src/utils/runCommand.js

```diff
@@ -10,8 +10,11 @@ const aioLogger = Logger('commerce:runCommand.js')
 export async function runCommand (command) {
   try {
     const { stdout, stderr } = await execPromise(command)
+    aioLogger.debug('stdout:', stdout)
+    aioLogger.debug('stderr:', stderr)
     return { stdout, stderr }
   } catch (error) {
-    aioLogger.debug(`Execution error: ${error}`)
+    aioLogger.debug(error)
+    throw error
```

Comment:
Since gh cli GQL errors throw and not stderr, and we were not re-throwing, the command could look like it worked when it did not.

PSA: Don't hide errors in catch blocks!!

Reply:
This change does cause an issue - the repo existence check throws if it does not exist. I am going to push a fix soon, at least for the github step.